dns-rc-autoscaler/nanny/main/pod_nanny.go

```diff
@@ -0,0 +1,91 @@
+/*
+Copyright 2016 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"errors"
+	"log"
+	"os"
+	"time"
+
+	flag "github.com/spf13/pflag"
+
+	"k8s.io/contrib/dns-rc-autoscaler/nanny"
+
+	client "k8s.io/kubernetes/pkg/client/clientset_generated/release_1_3"
+	"k8s.io/kubernetes/pkg/client/restclient"
+)
+
+var (
+	// Flags to identify the container to nanny.
+	podNamespace = flag.String("namespace", os.Getenv("MY_POD_NAMESPACE"), "The namespace of the ward. This defaults to the nanny pod's own namespace.")
+	podName      = flag.String("pod", os.Getenv("MY_POD_NAME"), "The name of the pod to watch. This defaults to the nanny's own pod.")
+	configFile   = flag.String("paramsFile", "", "Params file for the node scaling parameters")
+	verbose      = flag.Bool("verbose", false, "Turn on verbose logging to stdout")
+	// Flags to control runtime behavior.
+	pollPeriod = time.Millisecond * time.Duration(*flag.Int("poll-period", 10000, "The time, in milliseconds, to poll the dependent container."))
```

Comment:
Last week I think I saw the addon-resizer ignoring this flag. Can you confirm that this actually works?